common/app/http/Filters.scala

```diff
@@ -144,7 +144,6 @@ object Filters {
 
 }
 
-//Note: still using akka (instead of pekko) materializer from Play as both filters extend Play's Filter, so the types need to match
```

Comment:
Is it ok to just remove that comment now? I didn't dig deeper if something else needs to be done.